index.html

```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+            </div>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+    <section class="3-page">
+        <div class="snapshot2">
+            <a href=""><img src="Pictures/snapshot2.png" alt=""></a>
+            <h6>Multi-Post
+                Stories</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+
+
+    <section class="4-page">
+        <div class="snapshot2">
+            <a href=""><img src="Pictures/snapshot3.png" alt=""></a>
+            <h6>Tonic</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+    <section class="5-page">
+        <div class="snapshot2">
+            <a href=""><img src="Pictures/snapshot4.png" alt=""></a>
+            <h6>Multi-Post
+                Stories</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+
+
+    <section class="aboutme">
+        <div class="headline">
+            <div class="headline-heading">
+                <h1>About myself</h1>
```

Comment:
[OPTIONAL]

- [ ] It's better if the webpage has only one `h1` tag while the rest of the subheaders are `h2`, `h3`...